src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -41,6 +42,17 @@ public Version Version
             }
         }
 
+        public HttpVersionPolicy VersionPolicy
+        {
+            get { return _versionPolicy; }
+            set
+            {
+                CheckDisposed();
+
```

Comment:
```suggestion
```

Reply:
We have an empty line after `CheckDisposed` in every setter on this class, so I'm leaving it in. We can clean it up globally in separate PR if that's what we want.

Reply:
Ok